### ISSUE TITLE
[Feature Request] Bitwarden Integration

### ISSUE THREAD

**Issue by matvey-tk** [oliverschwendener/ueli#903]
Hello,

A Bitwarden integration of some kind would be massive (at least for me personally).

Thought I'd leave that here as a feature request in case you have spare time one day ;)

Functionality for searching existing records and generating a password would be great.

Thanks

**Comment by madsholme** [comment]
yes please, and also include lastpass. i would even pay for these kinds of integrations

**Comment by oliverschwendener** [comment]
Although I'm a big Bitwarden fan, I'm closing this feature request as I have no plans to implement this. If anyone else wants to do it, feel free to reopen.
